notebooks/automatic_embedding_tei_inference_endpoints.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.                "url": "ghcr.io/huggingface/text-embeddings-inference:0.5.0",
</pre><p>Maybe we should add a link somewhere to the list of Docker images? <a href="https://huggingface.co/docs/text-embeddings-inference/supported_models" target="_blank">https://huggingface.co/docs/text-embeddings-inference/supported_models</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>